### PR TITLE
RAT-435: checkstyle fixes for root package

### PR DESCRIPTION
**Pull request by Claudenw**
fix for RAT-435

### CODE REVIEW

File: apache-rat-core/src/main/java/org/apache/rat/BuilderParams.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public interface BuilderParams {
 
     /**
      * Gets one of the contained methods by name.
-     * 
+     *
```
Comment (ottlinger) [inline apache-rat-core/src/main/java/org/apache/rat/BuilderParams.java:36]: matcherMap() and licenseFamilies() are reported as never used - is there a test case missing or is it called by reflection as well, @Claudenw ?

File: apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java [hunk 2]
```diff
@@ -279,15 +279,15 @@ public void addExcludedCollection(StandardCollection collection) {
      * @param collection the StandardCollection to exclude.
      * @see ExclusionProcessor#addFileProcessor(StandardCollection)
      */
-    public void addExcludedFileProcessor(StandardCollection collection) {
+    public void addExcludedFileProcessor(final StandardCollection collection) {
```
Comment (ottlinger) [inline apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java:282]: What to do with unused methods in here ..... (possibly referenced in other PRs) @Claudenw

**Review by ottlinger — APPROVED** [review]
Apart from the unused methods in ReportConfiguration looks ok. Thanks

File: apache-rat-core/src/main/java/org/apache/rat/BuilderParams.java (hunk 1 above)
Comment (Claudenw) [reply apache-rat-core/src/main/java/org/apache/rat/BuilderParams.java:35]: I added comments to indicate the methods are called by reflection and also updated PR #366 (RAT-422) to test that the methods in the annotation exist in the BuilderParams class.

File: apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java (hunk 2 above)
Comment (Claudenw) [reply apache-rat-core/src/main/java/org/apache/rat/ReportConfiguration.java:282]: Handled in RAT-423

**Pull request merged**